x-pack/plugins/observability_solution/observability_onboarding/e2e/cypress/e2e/logs/system_logs.cy.ts

```diff
@@ -5,7 +5,8 @@
  * 2.0.
  */
 
-describe('[Logs onboarding] System logs', () => {
+// Failing 9.0 version update: https://github.com/elastic/kibana/issues/192624
+describe.skip('[Logs onboarding] System logs', () => {
```

Comment:
Due to https://github.com/elastic/package-registry/issues/1226.  @elastic/obs-ux-logs-team @elastic/obs-ux-onboarding-team obs-ux-onboarding-team  are you okay with circling back on this post merge?

Reply:
I tested this manually, will leave a note for tracking in https://github.com/elastic/kibana/issues/192624